velox/docs/functions/spark/datetime.rst

```diff
@@ -337,3 +325,33 @@ These functions support TIMESTAMP and DATE input types.
     part of the 53rd week of year 2004, so the result is 2004. Only supports DATE type.
 
         SELECT year_of_week('2005-01-02'); -- 2004
+
+Different Behaviors Between Simple And Joda Date Formmaters
+--------------------------------
```

Comment:
I think --- needs to go all the way to cover the length of the title.

Reply:
Thanks. Fixed